packaging/packager_rpm.sh

```diff
@@ -12,30 +12,33 @@ dnf -y install python3 python3-coverage
 dnf -y install python3-pillow
 # pip dependencies (for dependencies not available as RPM)
 dnf -y install gcc libX11-devel libXtst-devel python3-devel libpng-devel python3-pip redhat-rpm-config
+
```

Comment:
This was meant to be a single block installing system and build dependencies so not sure if the empty line separates in some clearer way.